fossunited/chapters/web_template/foss_clubs_home_page/foss_clubs_home_page.html

```diff
@@ -0,0 +1,409 @@
+        letter-spacing: -0.045rem;
+
+    }
+
+    .btn-primary:active, .btn-primary:hover{
+        background-color: var(--Foundation-FOSS-Mint-foss-mint-600, #07A748) !important;
+        border:none !important;
+    }
+
+    .hero-image > img{
+        transform: rotate(-2deg) scale(1.8);
+        transform-origin: top;
+    }
+
+    .autocomp {
+        background: #ffffff;
+        border-radius: 0 0 5px 5px;
+        border: 1px solid #EAECF0;
+        border-top: 0;
+        box-shadow: 2px 2px 2px #f8f8f8;
+        text-align: left;
+    }
+    .autocomp-item {
+        padding-bottom: 5px;
+        padding: 10px;
+        cursor: pointer;
+    }
+    .autocomp-item:hover,
+    .autocomp-sel {
+        background: #f8f8f8;
+        font-weight: bold;
+        color: var(--Foundation-FOSS-Mint-foss-mint-600, #07A748);
+    }
+
+    @media (max-width: 768px){
+        .club-hero-section{
+            flex-direction: column;
+            gap: 2.5rem;
+        }
+
+        .hero-container{
+            padding-left: 0.5rem;
+            padding-right: 0.5rem;
+            gap: 3rem;
+        }
+
+        .hero-header{
+            max-width: 100%;
+            display: flex;
+            flex-direction: column;
+        }
+
+        .hero-header-heading{
+            align-items: center;
+        }
+
+        h1{
+            font-size: 2rem;
+            letter-spacing: -0.12rem;
+        }
+
+        .hero-image{
+            height: 10rem;
+            overflow: hidden;
+        }
+        .hero-image > img{
+            transform: rotate(26deg) scale(1.45);
+            transform-origin: 50% 42%;
+        }
+
+        .hero-club-search{
+            max-width: 100%;
+        }
+
+        #search-club::placeholder{
+            font-size: 1rem;
+        }
+
+        .btn-primary{
+            width: 100%;
+        }
+
+    }
+</style>
+
+
+<div class="club-hero-section container">
+
+    <div class="hero-container container">
+
+        <div class="hero-header">
+
+            <div class="hero-header-heading">
+                <img src="assets/fossunited/images/chapter/fossclub_logo.svg" alt="">
+                <h1>FOSS <span>Clubs</span></h1>
+            </div>
+
+            <p>{{subtext}}</p>
+
+        </div>
+
+
+        <div class="hero-search">
+
+                <h5>{{subheading}}</h5>
+                <div class="hero-club-search">
+
+                    <input type="text" id="search-club" placeholder="Search by name of club or city">
+
+                </div>
+
+
+        </div>
+
+        <div class = "hero-cta">
+
+            <div class="button-container">
+
+                    <button class="btn-primary" href="{{cta_link}}">{{cta_text}}
+                    </button>
+
+            </div>
+
+        </div>
+    </div>
+
+    <div class="hero-image">
+        <img src="assets/fossunited/images/illustrations/city-communities.svg" alt="">
+    </div>
+
+    <script>
+
+        function autocomp(el, options = {}) {
```

Comment:
Let's figure out a way to move this in next PR

Reply:
Yes, need to - I wasn't able to get it fixed